src/validation.cpp

```diff
@@ -554,6 +554,16 @@ class MemPoolAccept
     */
     PackageMempoolAcceptResult AcceptMultipleTransactions(const std::vector<CTransactionRef>& txns, ATMPArgs& args) EXCLUSIVE_LOCKS_REQUIRED(cs_main);
 
+    /**
+     * Submission of a subpackage.  If only 1 transaction exists in subpackage, calls
+     * AcceptSingleTransaction() with adjusted ATMPArgs to avoid additional package policy
+     * restrictions like PackageMempoolChecks() and disabled RBF, and creates a
```

Comment:
The “disabled RBF” can sound as confusing. There is no nsequence opted-out of the transaction candidate targeted by a replacement, it sounds rather to mean the `m_allow_replacement` of `ATMPArgs` which is true for `SingleInPackageAccept` and false for `PackageChildWithParents`.

Reply:
Edited comment